<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Personnel</string>
     </attribute>
     <widget class="QTabWidget" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Prestige Elite Std</family>
        <pointsize>9</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Prestige Elite Std&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_43">
       <attribute name="title">
        <string>Ajouter personnel</string>
       </attribute>
       <widget class="QLabel" name="label_51">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Identifiant&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_63">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_65">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>190</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>130</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>190</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_personnel">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>310</y>
          <width>94</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #060606; }
QComboBox{
border-style: solid;
border: 2px solid black;


 }</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_67">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Age&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>220</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_service">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>250</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_70">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>250</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant service</string>
        </property>
       </widget>
       <widget class="QLabel" name="pe1">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>1081</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>pe1</zorder>
       <zorder>label_51</zorder>
       <zorder>label_63</zorder>
       <zorder>label_65</zorder>
       <zorder>lineEdit_id</zorder>
       <zorder>lineEdit_nom</zorder>
       <zorder>lineEdit_prenom</zorder>
       <zorder>ajouter_personnel</zorder>
       <zorder>label_67</zorder>
       <zorder>comboBox</zorder>
       <zorder>lineEdit_id_service</zorder>
       <zorder>label_70</zorder>
      </widget>
      <widget class="QWidget" name="tab_44">
       <attribute name="title">
        <string>Afficher personnel</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1451</width>
          <height>521</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_45">
         <attribute name="title">
          <string>afficher</string>
         </attribute>
         <widget class="QTableView" name="tabpersonnel">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>1441</width>
            <height>421</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:black;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="exporter_pers">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>20</y>
            <width>94</width>
            <height>30</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #060606; }
QComboBox{
border-style: solid;
border: 2px solid black;


 }</string>
          </property>
          <property name="text">
           <string>Exporter PDF</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="txt_pers">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>170</y>
            <width>811</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_46">
         <attribute name="title">
          <string>afficher_tri</string>
         </attribute>
         <widget class="QTableView" name="tab_tri">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>1</y>
            <width>1441</width>
            <height>481</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_47">
         <attribute name="title">
          <string>rechercher</string>
         </attribute>
         <widget class="QTableView" name="tableView_3">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>10</y>
            <width>981</width>
            <height>481</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_71">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>140</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Identifiant</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_id_4">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>140</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: black;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="rechercher_personnel">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>230</y>
            <width>94</width>
            <height>30</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #060606; }
QComboBox{
border-style: solid;
border: 2px solid black;


 }</string>
          </property>
          <property name="text">
           <string>rechercher</string>
          </property>
         </widget>
         <widget class="QLabel" name="pe2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-50</y>
            <width>1081</width>
            <height>531</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <zorder>pe2</zorder>
         <zorder>tableView_3</zorder>
         <zorder>label_71</zorder>
         <zorder>lineEdit_id_4</zorder>
         <zorder>rechercher_personnel</zorder>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_48">
       <attribute name="title">
        <string>Supprimer personnel</string>
       </attribute>
       <widget class="QLabel" name="label_72">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>150</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="tmppersonnel">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_personnel">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>250</y>
          <width>94</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #060606; }
QComboBox{
border-style: solid;
border: 2px solid black;


 }</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="pe3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>1081</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>pe3</zorder>
       <zorder>label_72</zorder>
       <zorder>tmppersonnel</zorder>
       <zorder>supprimer_personnel</zorder>
      </widget>
      <widget class="QWidget" name="Modifier">
       <attribute name="title">
        <string>Modifier personnel</string>
       </attribute>
       <widget class="QLabel" name="labelm">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2m">
        <property name="geometry">
         <rect>
          <x>206</x>
          <y>160</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3m">
        <property name="geometry">
         <rect>
          <x>206</x>
          <y>190</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idm">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomm">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_premomm">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_persoonel">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>290</y>
          <width>94</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Prestige Elite Std</family>
          <pointsize>-1</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #060606; }
QComboBox{
border-style: solid;
border: 2px solid black;


 }</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_agem">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>220</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9m">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>220</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_37">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>260</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_73">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>260</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant service</string>
        </property>
       </widget>
       <widget class="QLabel" name="pe4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>1081</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>pe4</zorder>
       <zorder>labelm</zorder>
       <zorder>label_2m</zorder>
       <zorder>label_3m</zorder>
       <zorder>lineEdit_idm</zorder>
       <zorder>lineEdit_nomm</zorder>
       <zorder>lineEdit_premomm</zorder>
       <zorder>modifier_persoonel</zorder>
       <zorder>lineEdit_agem</zorder>
       <zorder>label_9m</zorder>
       <zorder>lineEdit_37</zorder>
       <zorder>label_73</zorder>
      </widget>
      <widget class="QWidget" name="tab_49">
       <attribute name="title">
        <string>Verifer personnel</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_38">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border-color: red;
border: 2px solid gray;
border-radius: 8px;
border-color: black;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_74">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="verifier_personnel">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>240</y>
          <width>94</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #060606; }
QComboBox{
border-style: solid;
border: 2px solid black;


 }</string>
        </property>
        <property name="text">
         <string>Verifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="pe5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>1081</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>pe5</zorder>
       <zorder>lineEdit_38</zorder>
       <zorder>label_74</zorder>
       <zorder>verifier_personnel</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
